test/jinx_test.exs

```diff
@@ -1,8 +1,49 @@
 defmodule JinxTest do
   use ExUnit.Case
-  doctest Jinx
+  import Jinx, only: [json_for: 2, json_for: 3]
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  describe "json_for" do
+    test "it properly adds fields to attributes" do
+      attrs = %{foo: "bar", food: "candy bar"}
+      json_payload = json_for(:foo, attrs)
+
+      expected = %{
+        "data" => %{"attributes" => attrs, "type" => "foo"},
+        "format" => "json-api"
+      }
+
+      assert json_payload == expected
+    end
+
+    test "it properly adds relationships to data" do
+      attrs = %{name: "Scott", isTheBest: true}
+      relationships = %{"job" => %{"data" => %{type: "job", id: 1}}}
+
+      json_payload = json_for(:person, attrs, relationships)
+
+      expected = %{
+        "data" => %{
+          "attributes" => attrs,
+          "type" => "person",
+          "relationships" => relationships
+        },
+        "format" => "json-api"
+      }
+
+      assert json_payload == expected
+    end
+
+    test "it replaces_snake case with kebab-case for json type" do
+      json_payload = json_for(:some_super_cool_model, %{})
+
+      assert Map.get(json_payload["data"], "type") == "some-super-cool-model"
+    end
+
+    test "it removes __meta__ and __struct__ attrs" do
+      attrs = %{:foo => "bar", :__meta__ => "stuff", :__struct__ => "moar stuff"}
+      data =  json_for(:foo, attrs) |> Map.get("data")
+
+      assert Map.get(data, "attributes") == %{:foo => "bar"}
```

Comment:
This is super minor but just for consistency, I think I like how you structured this in the previous [test](https://github.com/SViccari/jinx/pull/1/files#diff-43a0d71f7e1fba401f4fc4f64f262acbR39). Although, I don't feel strongly about it, I just like consistency so my brain doesn't try to figure out if there is a reason for the different approach.